eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -2597,17 +2598,78 @@ public function testMoveSubtreeHidden()
         // Load new parent location
         $newParentLocation = $locationService->loadLocation($demoDesignLocationId);
 
+        // Move location from "Home" to "Demo Design" (not container)
+        $locationService->moveSubtree($locationToMove, $newParentLocation);
+    }
+
+    /**
+     * Test for the moveSubtree() method.
+     *
+     * @see \eZ\Publish\API\Repository\LocationService::moveSubtree()
+     * @depends eZ\Publish\API\Repository\Tests\LocationServiceTest::testLoadLocation
+     * @expectedException \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
```

Comment:
This is deprecated and won't work on `ezplatform-kernel:1.x`. Use `$this->expectException` instead please, but be careful when placing it - it should be located just before the call which is supposed to throw that exception.